examples/medplum-demo-bots/src/billing-bots/cms-1500.ts

```diff
@@ -0,0 +1,381 @@
+      `${employmentImpactedStart} - ${employmentimpactedEnd}`,
+    '17,Name of Referring Provider or Other Source,Full name of referring provider or other source,' + referrerName,
+    '17a,Referring Provider Identifier,Identifier of the referring provider,',
+    '17b,Referring Provider NPI,NPI of the referring provider,' + referrerNpi,
+    '18,Hospitalization Dates Related to Current Services,Dates of hospitalization related to current services,' +
+      `${hospitalizationStart} - ${hospitalizationEnd}`,
+    '19,Additional Claim Information,Additional claim information,',
+    '20,Outside Lab?,Indicates if outside lab services were used,' + outsideLab,
+    '20,Laboratory Charges,Charges for laboratory services,' + outsideLabCharges,
+    '21,Diagnosis or Nature of Illness or Injury,Diagnosis codes,' + diagnosis,
+    '22,Resubmission Code,Resubmission code,' + resubmissionCode,
+    '22,Original Reference Number,Original reference number,' + originalReference,
+    '23,Prior Authorization Number,Prior authorization number,' + priorAuthRefNumber,
+    '24A,Date(s) of Service,Dates of service,' + dateOfService,
+    '24B,Place of Service,Place of service,' + placeOfService,
+    '24C,EMG,EMG indicator,' + emergency,
+    '24D,Procedures, Services, or Supplies,Procedure codes,' + procedureCode,
+    '24D,Procedures, Services, or Supplies,Modifiers,' + modifiers,
+    '24E,Diagnosis Pointer, Diagnosis pointer,' + diagnosisPointer,
+    '24F,Charges,Charges for service,' + charges,
+    '24G,Days or Units,Number of days or units,' + daysOrUnits,
+    '24H,EPSDT Family Plan,EPSDT family plan indicator,' + familyPlanIndicator,
+    '24I,ID Qualifier,ID qualifier,',
+    '24J,Rendering Provider ID #,Rendering provider ID number,',
+    '25,Federal Tax ID Number,Federal tax ID number,' + fedTaxNumber,
+    '25,Federal Tax ID Type,Federal tax ID type,' + fedTaxType,
+    "26,Patient's Account Number,Patient's account number," + patientAccountNumber,
+    '27,Accept Assignment?,Indicates if provider accepts assignment,',
+    '28,Total Charge,Total charge for services,' + totalCharge,
+    '29,Amount Paid,Amount paid,' + patientPaid,
+    '30,Rsvd for NUCC Use,Reserved for NUCC use,',
+    '31,Signature of Physician or Supplier,Signature of physician or supplier,',
+    '32,Service Facility Location Information,Location of service facility,' + serviceLocation,
+    '32a,Service Facility NPI,NPI of service facility,' + serviceNpi,
+    '32b,Other ID #,Other ID number of service facility,',
+    '33,Billing Provider Info & Ph #,Billing provider information and phone number,' +
+      billingLocation +
+      billingPhoneNumber,
+    '33a,Billing Provider NPI,NPI of billing provider,' + providerNpi,
+    '33b,Other ID #,Other ID number of billing provider,',
+  ];
+
+  let cms1500Text: string = '';
+
+  for (const line of cms1500Lines) {
+    cms1500Text += line + '\n';
+  }
+
+  console.log(cms1500Text);
+  return cms1500Text;
```

Comment:
We should save this as Document (or something) and link to patient and Encounter right?  @rahul1 - what's the right spot?